pyvista/core/composite.py

```diff
@@ -1110,6 +1110,8 @@ def as_polydata_blocks(self, copy=False):
             if block is not None:
                 if isinstance(block, MultiBlock):
                     dataset.replace(i, block.as_polydata_blocks(copy=copy))
+                elif isinstance(block, pyvista.PointSet):
+                    dataset.replace(i, block.cast_to_polydata(deep=copy))
                 elif not isinstance(block, pyvista.PolyData):
                     dataset.replace(i, block.extract_surface())
```

Comment:
I'm surprised this didn't handle it 😕

Reply:
It kinda makes sense. `PointSet` does not have a "surface" per se.

Reply:
Hm. Should we raise an error in that filter for PointSets like other geometry/cell-wise filters? This filter seems like a no-op